CIScripts/ContrailUtils.ps1

```diff
@@ -0,0 +1,92 @@
+function Get-AccessTokenFromKeyStone {
+    Param ([Parameter(Mandatory = $true)] [string] $AuthUrl,
+        [Parameter(Mandatory = $true)] [string] $TenantName,
```

Comment:
Wrong indentation here (in reference to our other PS function)

Reply:
Ok I will change that